README.md

```diff
@@ -217,8 +217,21 @@ For the digital signature of your documents you need a signing certificate in .p
 
 Want to create a production ready docker image? Follow these steps:
 
-- Run `./docker/build.sh` in the root directory.
-- Publish the image to your docker registry of choice.
+- cd into `docker` directory
+- Make `build.sh` executable by running `chmod +x build.sh`
```

Comment:
It should already be executable 🤔 Git preserves the permission bits afaik

Reply:
Not sure it fully does. Often faced similar issues on Linux.

[Ref](https://archive.kernel.org/oldwiki/git.wiki.kernel.org/index.php/ContentLimitations.html)

Reply:
> except for the executable bit

But if you're having issues it's probably good to include anyways!

Reply:
Great, Thank you 😄